packages/protocol/schemaCodegen.cjs

```diff
@@ -18,7 +18,7 @@ tsGen.DEFAULT_OPTIONS.bannerComment = `
 
 const tsOut = path.resolve('./src/data/form.ts');
 var schemaUri =
-  'https://jenkins.ivyteam.io/job/core_json-schema/job/master/lastSuccessfulBuild/artifact/workspace/ch.ivyteam.ivy.dialog.form.lsp.schema/target/schema/form/13.1.0/form-tsgen.json';
+  'https://jenkins.ivyteam.io/job/core_json-schema/job/add-confirm-dialog-to-button/1/artifact/workspace/ch.ivyteam.ivy.dialog.form.lsp.schema/target/schema/form/13.1.0/form-tsgen.json';
```

Comment:
revert. by the way you don't have to do this, just run the command with the URL as parameter